src/main/java/com/carrotcreative/recyclercore/inject/InjectController.java

```diff
@@ -0,0 +1,18 @@
+package com.carrotcreative.recyclercore.inject;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Created by kaushiksaurabh on 10/1/15.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface InjectController
```

Comment:
we can make two different interfaces, but the Layout is coupled with the `Controller`, one cannot exist without the other, so maybe we should not ?